### PR TITLE
Get ready to test this repo on travis

### PR DESCRIPTION
**Pull request by jzoldak**
@mjstallard @benmadrone developing here until we get travis set up on the upstream repo.

### CODE REVIEW

**Comment by benpatterson** [comment]
@jzoldak I was going to use my @benmadrone profile if that's alright... I'll comment from there.

**Comment by benmadrone** [comment]
@jzoldak 

My only suggestion is adding the pep8 command to the README for folks that want to contribute.

File: dev-requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+pep8==1.7.0
```
Comment (benmadrone) [inline dev-requirements.txt]: Not a must, but you could

```
-r requirements.txt

pep8==1.7.0
```

**Comment by jzoldak** [comment]
@benmadrone cool, thx for the review. I'll merge this now, let's add a makefile and update the README in the next PR.

**Pull request merged**